### ISSUE TITLE
Coding Platform

### ISSUE THREAD

**Issue by littleironical** [inaxia/hashcode_web#5]
In the Coding page, we are using Repl.it for now, but it requires Sign-in to run any code.
And we don't want that.

So, there are two ways to resolve this:

- Either you can find a Coding Platform that doesn't require you to sign-in to run your code, but this may not compile every coding languages that we have in our list #3. So, you can find different coding platforms for different coding languages.
- Or you can find a coding platform that compiles every coding languages in our list and also doesn't require sign-in

**Comment by virti313** [comment]
I can try looking for a solution to fix this issue. Please assign it to me if possible.

**Comment by littleironical** [comment]
Yes @virti313 you can, I hope you've understood the issue properly. Assigning this to you.

**Comment by virti313** [comment]
https://www.onlinegdb.com/online_c_compiler 
https://ideone.com/
Kindly check these links out and let me know if this is the solution you were looking for.

**Comment by littleironical** [comment]
Both the platform are good @virti313, but use [OnlineGDB](https://www.onlinegdb.com/online_c_compiler) as it has all the compilers available.
Now you have to replace **Repl.it** with **OnlineGDB** in `views/index.ejs` folder. And do changes in **Develop** branch.
